runtime/rococo/src/lib.rs

```diff
@@ -715,19 +708,14 @@ impl paras_registrar::Config for Runtime {
 	type WeightInfo = paras_registrar::TestWeightInfo;
 }
 
-/// An insecure randomness beacon that uses the parent block hash as random material.
-///
-/// THIS SHOULD ONLY BE USED FOR TESTING PURPOSES.
-pub struct ParentHashRandomness;
```

Comment:
This is the exception in this PR, Randomness is now derived from Babe only, as in westend.